### PR TITLE
Minor updates to mariadb/README.md

### PR DESCRIPTION
**Pull request by LeamHall**
Changed ":tag" to ":latest".
Added user to --env option usage.
Added some notes on cleanup.

### CODE REVIEW

**Comment by LeamHall** [comment]
Updated mariadb/content.md, once DBlack pointed me to the right file to edit.

File: mariadb/README.md [hunk 1]
```diff
@@ -63,30 +63,64 @@ The intent is also to maintain high compatibility with MySQL, ensuring a library
...
+
+```console
+$ docker ps
+```
+
+To get the IP address of your server instance, use the container ID given above:
+
+```console
+$ docker inspect <server container ID> |grep IPAddress
+```
```
[the remark this answers is not in the document]
Comment (grooverdan) [reply mariadb/README.md:92]: Looks good. To keep the CI tools happy README.md needs to be left out of the changes.

**Comment by tianon** [comment]
Just did the rebase/force push dance:

```console
$ git pull --rebase https://github.com/docker-library/docs.git master # rebase to the latest upstream commit
$ git rebase -i FETCH_HEAD # squash all the commits down to just one
$ git show FETCH_HEAD:mariadb/README.md > mariadb/README.md # revert README.md changes
$ git add mariadb/README.md # stage the change
$ git commit --amend # add the change to the commit
$ git push -f # force push back to the branch to update the PR
```

Thanks for working through this!

**Comment by LeamHall** [comment]
Guys, thank you for your expertise!

**Pull request merged**